apps/lead/models.py

```diff
@@ -223,6 +223,10 @@ def get_for(cls, user):
             assessment_id=models.F('assessment'),
             verified_entries_count=models.Count('entry',
                                                 filter=models.Q(entry__verified=True)),
+            organization_types=models.functions.Coalesce(
+                'authors__parent__organization_type',
+                'authors__organization_type'
+            ),
```

Comment:
Let's move this to the custom filter function. Wouldn't make sense to calculate this when not required.